src/app/components/ContactViewProperty.js

```diff
@@ -105,11 +105,20 @@ const ContactViewProperty = ({ property, properties, contactID }) => {
         }
     };
 
+    const applyEllipsisClass = () => {
+        switch (field) {
+            case 'note':
+                return null;
+            default:
+                return 'ellipsis';
+        }
+    };
+
     return (
         <Row>
             <ContactLabelProperty field={field} type={type} first={first} />
             <div className="flex flex-nowrap flex-items-center w100">
-                <span className="mr0-5 ellipsis">{getContent()}</span>
+                <span className={'mr0-5 flex-item-fluid ' + applyEllipsisClass()}>{getContent()}</span>
```

Comment:
- Use `classnames` from component helper
- Add the condition in the `Array` itself:
```
<span className={classnames(['mr0-5 flex-item-fluid', !['note'].includes(field) && 'ellipsis'])}>{getContent()}</span>
```

Reply:
Done with https://github.com/ProtonMail/proton-contacts/pull/101/commits/9c5b219439d415a040c72e42d0a8d485d184baae